### PR TITLE
examples: use optee_examples instead of hello_world rep

### PR DESCRIPTION
**Pull request by igoropaniuk**
Relates to https://github.com/linaro-swg/optee_examples/pull/1, https://github.com/OP-TEE/manifest/pull/69, https://github.com/OP-TEE/build/pull/170
`Signed-off-by: Igor Opaniuk <igor.opaniuk@linaro.org>`

### CODE REVIEW

File: unshallow.sh [hunk 1]
```diff
@@ -17,8 +17,8 @@ git fetch github --unshallow
 echo "unshallow client"
 cd ~/work/swg/svp/hikey-n-4.9/external/optee_client
 git fetch github --unshallow
-echo "unshallow hello world"
-cd ~/work/swg/svp/hikey-n-4.9/external/optee_hello_world
+echo "unshallow optee_examples"
```
Comment (jockebech) [inline unshallow.sh:20]: I've never really seen this "unshallow.sh" script before. To me it looks like a helper script for some users. It even has hardcoded paths. But you changes here should be fine of course.

**Comment by jockebech** [comment]
`Acked-by: Joakim Bech <joakim.bech@linaro.org>`

File: unshallow.sh (hunk 1 above)
Comment (vchong) [reply unshallow.sh:20]: @jbech-linaro Yes sorry this is for my own use when repo syncing with depth 1 and unshallowing the repos we usually touch before patches can be applied. I can send in a PR later to make the patch relative to pwd.

**Comment by vchong** [comment]
`Acked-by: Victor Chong <victor.chong@linaro.org>`

**Comment by igoropaniuk** [comment]
Applied tags and rebased. Ready for merge!

**Comment by vchong** [comment]
Merged. Thanks!

**Pull request merged**